src/django_components_js/src/errorHandling.ts

```diff
@@ -0,0 +1,40 @@
+import { isArray, isFunction, isPromise } from './utils';
```

Comment:
This file contains code for safe calling of functions, seen in Vue and Alpine. It's just a boilerplate to make sure that when we run JS code for individual components, that the process continues even if the user-provided JS throws an error.